pkg/cmd/step/verify/step_verify_preinstall.go

```diff
@@ -172,6 +173,11 @@ func (o *StepVerifyPreInstallOptions) Run() error {
 	}
 	log.Logger().Info("\n")
 
+	err = o.installMissingDependencies()
```

Comment:
do we really want to install or just say they are not installed? I guess what bugs me is that this is supposed to be a _verify_, not an install step.

Reply:
I think the fact that they are an internal implementation detail for JX now that they are in ~/.jx/bin would make it ok to auto install. Is there a better location to do this you would suggest that would remove it out of the verify? Ideally longer terms we'd remove the need to call out to external binaries.

Reply:
The alternative is to fail with an error and let the user fix it. Given that we want to install into JX_HOME/bin it would not make much sense to just display `jx install dependencies`, so I guess it is the best we can do for now.